li2mni/qc1.py

```diff
@@ -0,0 +1,157 @@
+                                  .convert('RGB'))
+                    titles.append(f)
+            if pattern == "T1wLi":
+                if f.endswith('.png'):
+                    images.append(Image.open(os.path.join(png_folder, f))
+                                  .convert('RGB'))
+                    titles.append(f)
+        # # Create the font and size for the title
+        font = ImageFont.truetype("/usr/share/fonts/truetype/ubuntu/"
+                                  "Ubuntu-B.ttf", size=70)
+
+        # Add the title to each image
+        for i, image in enumerate(images):
+            # Create a new ImageDraw object
+            draw = ImageDraw.Draw(image)
+            # Set the position and text for the title
+            title = titles[i]
+            x = 20
+            y = 20
+            # Draw the title on the image
+            draw.text((x, y), title, font=font, fill=(0, 0, 0))
+            # Save the image as a PDF page
+            image.save(tmpdir+'/page{}.pdf'.format(i+1), 'PDF',
+                       resolution=100.0)
+        # Merge the PDF pages into a single file
+        pdf_pages = [convert_from_path(os.path.join(tmpdir, i))[0]
+                     for i in os.listdir(tmpdir)
+                     if i.endswith(".pdf")]
+        pdf_pages[0].save(pdf_output, 'PDF', resolution=100.0, save_all=True,
+                          append_images=pdf_pages[1:])
+    df_qc = pd.DataFrame({"sub": titles, "qc": 1})
+    df_qc.to_csv(pdf_output.replace(".pdf", ".csv"), sep="\t", index=False)
+
+
+def make_png(anatomical, outdir, pattern="T1wli"):
+    im = nibabel.load(anatomical)
+    arr = im.get_fdata()
+
+    outfile = os.path.join(outdir, f"{pattern}.png")
+    plotting.plot_anat(im, display_mode="z",
+                       cut_coords=25, black_bg=True,
+                       output_file=outfile)
+    arr = plt.imread(outfile)
+    cut = int(arr.shape[1] / 5)
+    plt.figure(figsize=(10, 10))
+    arr = np.concatenate(
+        [arr[:, idx * cut: (idx + 1) * cut] for idx in range(5)], axis=0)
+    plt.imshow(arr)
+    plt.axis("off")
+    plt.savefig(outfile, dpi=300)
+    plt.close()
+
+
+def get_anat(path):
+    with open(path) as flux:
+        lignes = flux.readlines()
+        for i in range(len(lignes)):
+            lignes[i] = lignes[i].replace("\n", "")
+    return lignes
+
+
+def psc2_to_psc1(psc2, df):
+    return df[df["psc2"] == int(psc2)]["psc1"].values[0]
+
+
+def all(list_nii, outdir, pattern="T1wli", skip_png=False, transcoding=None,
+        skip_pdf=False):
+    """ Launch the lithium rawdata quality control workflow.
+
+    Parameters
+    ----------
+    list_nii: str
+        path to a .txt with one line per .nii.gz
+    pattern: str
+        should be Li or T1wLi
+    outdir: str
+        path to the destination folder.
+
+    Optionnal
+    ---------
+    skip_png: bool
+        skip png creation step.
+    skip_pdf: float
+        skip pdf creation step.
+    transcoding: str
```

Comment:
you can't use the transcoding file (in a safe). Use the participants.tsv file instead.